paper.tex

```diff
@@ -476,137 +476,170 @@ \section*{Appendix}
 during that time period. Individuals can move between populations
-either through ancestor-descendant relationships between the populations,
-or through processes of continuous or pulse migration.
-No other properties of the populations are specified in the model:
-we are concerned only with defining the populations, their sizes, and the
+either according to their ancestor-descendant relationships
+or through processes involving migrations.
+Few other properties of the populations are specified in the model:
+we are concerned primarily with defining the populations, their sizes, and the
 movement of individuals between those populations.
+\aprcomment{previous sentence changed to allow for selfing/cloning}
+
+\subsection{Time units}
 
 Population and event times are written as units in the past, so that time zero
 corresponds to the final generation or ``now'', and event times in the past are
-values greater than zero with larger values for events that occur in the more
+values greater than zero with larger values corresponding to times in the more
 distant past. By having time units increase into the past, we avoid the need to
-choose an arbitrary point in history as ``time zero'', so that instead zero
-refers to ``now'' or the final generation of a simulation. A natural
+choose an arbitrary point in history as ``time zero''. A natural
 specification for time units is in generations, although other time units are
 permitted, such as years, accompanied by the generation time.
 
+\subsection{Sizes and epochs}
+
 Population sizes are given as numbers of individuals, and details
 such as ploidy levels are considered external to the model.
-We therefore focus on the number of individuals as opposed to the number of genomes.
+We therefore focus on the number of individuals as opposed
+to the number of genome copies.
 Sizes and mating system details are specified for each population within
 epochs.
 Epochs are contiguous time intervals that define
 the existence interval of the population. Each epoch specifies the population size
 over that interval, which can be a constant value or function defined by start
 and end sizes that must remain positive.
 
-\krtcomment{The paragraph(s) below subsume(s)/replace(s) some of what appears in later pargraphs.
-	I'm not making an attempt yet to revise that later content--I think other authors should
-	weigh in.}
+%\krtcomment{The paragraph(s) below subsume(s)/replace(s) some of what appears in later pargraphs.
+%	I'm not making an attempt yet to revise that later content--I think other authors should
+%	weigh in.}
+
+%\krtcomment{Aaron -- I have perhaps over-simplified the description of marginal allele frequency dynamics?}
 
-\krtcomment{Aaron -- I have perhaps over-simplified the description of marginal allele frequency dynamics?}
+\subsection{Population dynamics}
 
 Within a population, we implicitly assume that allele frequency dynamics can
```

Comment:
I think we should delete "implicitly" here - this is what we assume and we should be explicit about it.